### DISCUSSION TITLE
Cmd problem

### DISCUSSION THREAD

**Discussion by thegreatestbrian** [Melledy/Grasscutter#1183] [Q&A]
That's weird but how to fix it?
![Screenshot 2022-06-03 220830](https://user-images.githubusercontent.com/105790655/171883412-4ec201de-3e5f-445b-9636-9d5a6ff0c5b0.png)

**Comment by errqww** [comment]
i think you should enter the entire path of your desktop,as cd c:/users/username/desktop

**Reply by errqww** [reply]
current cd means to enter the file in the current path
